website/content/docs/security/acl/acl-rules.mdx

```diff
@@ -600,6 +611,30 @@ operator = "read"
 
 </CodeTabs>
 
+## Peering Rules
+The `peering` resource controls access to cluster peerings in the [Cluster Peering API](/api-docs/peering).
```

Comment:
@freddygv : do any built-in policies have `peering` resource permissions?

For example, I know `namespace-management` wouldn't. Is there an equivalent built-in for partitions that would?

Reply:
It's only embedded in the global management policy

There's no built-in partition management policy at the moment